examples/bow/README.md

```diff
@@ -0,0 +1,37 @@
+# Bag of Word Model Generator
+
+## Overview
+
+Bag-of-Word (BoW) is a mechanism used in Natural Language Processing (NLP) to extract features in documents and texts. This example implements a simple BoW model using Ballerina I/O operations.
+
+## Implementation
+
+The input for this program is a raw text file. The program reads the given raw-text file line by line as a stream using the following I/O API.
+
+```ballerina
+stream<string, io:Error?> lineStream = check io:fileReadLinesAsStream("./resources/data.txt");
+```
+
+Then process each line in the stream to generate the BoW model.
+
+## Run the Example
+
+First, clone this repository, and then run the following commands to run the example on your local machine.
+
+```sh
+$ cd examples/bow
+$ bal run
+```
+
+Then provide the input value. This input value indicates the number of most frequent words users wants to retrieve. For example:
+
+```
+> Enter the frequency boundry value: 10
```

Comment:
Is `frequency boundry value` a term specific to the BoW model? If not can we say like `Enter the number of most frequent words to retrieve: `

Reply:
Sounds good. Fixed in https://github.com/ballerina-platform/module-ballerina-io/pull/447/commits/7c5853ff449370f3c48ff599eb374370e158ac86